cppsdk/source/playfab/PlayFabEventRouter.cpp

```diff
@@ -0,0 +1,24 @@
+#include <stdafx.h>
+
+#ifndef DISABLE_PLAYFABENTITY_API
+
+#include <playfab/PlayFabEventRouter.h>
+
+namespace PlayFab
+{
+    PlayFabEventRouter::PlayFabEventRouter()
+    {
+        //this->pipelines.emplace(0, std::move(std::make_shared<PlayFabPipeline>(std::make_shared<PlayFabPipelineSettings>()))); // add PlayFab pipeline by default
+    }
+
+    void PlayFabEventRouter::RouteEvent(std::shared_ptr<const IPlayFabEmitEventRequest> request) const
```

Comment:
Is this async ?

Reply:
Yes, it will be. More specifically, not blocking.

---
In reply to: [227948732](https://github.com/PlayFab/XPlatCppSdk/pull/6#discussion_r227948732) [](ancestors = 227948732)